### DISCUSSION TITLE
Please help me to foods

### DISCUSSION THREAD

**Discussion by e061u** [NAIST-SE/AttractingContributors#21] [General]
Please help me to foods. Please check my favorite foods on README.

**Comment by 278Mt** [comment]
added! and plz help my favorite movies!
https://github.com/278mt/AttractingContributors
thx!

**Comment by manzoku23** [comment]
Added my favorite one.
Please help me
https://github.com/Exe-dev/AttractingContributors-POKEMON

**Comment by sergiodl-naist** [comment]
Added one of my favorite ones
Please visit and try to ask a question or recommend one vtuber on my fork!
https://github.com/sergiodl-naist/AttractingContributors-VTubers

**Comment by sergiodl-naist** [comment]
added one of my favorites
Please visit and try to ask a question or recommend one vtuber on my fork!
https://github.com/sergiodl-naist/AttractingContributors-VTubers

Also, try editing your post to include your repository URL! It will be easier for others to find it 
https://github.com/iexuxs-m/AttractingContributors-Foods
